src/resources/i18n-target.html

```diff
@@ -0,0 +1,52 @@
+<template>
+
+  <button id="locale-changer-de" click.delegate="changeLocale('de')">DE</button>
+
+  <span id="i18n-simple" t="simple.text;[title]simple.attr"></span>
+
+  <span id="i18n-ctx" t="status"></span>
+  <span id="i18n-ctx-dispatched" t="status" t-params.bind="{context: 'dispatched', date: dispatchedOn}"></span>
+  <span id="i18n-ctx-delivered" t="status" t-params.bind="{context: 'delivered', date: deliveredOn}"></span>
+
+  <span id="i18n-items-0" t="itemWithCount" t-params.bind="{count: 0}"></span>
+  <span id="i18n-items-1" t="itemWithCount" t-params.bind="{count: 1}"></span>
+  <span id="i18n-items-10" t="itemWithCount" t-params.bind="{count: 10}"></span>
+
+  <span id="i18n-interval-0" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 0}"></span>
+  <span id="i18n-interval-1" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 1}"></span>
+  <span id="i18n-interval-2" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 2}"></span>
+  <span id="i18n-interval-3" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 3}"></span>
+  <span id="i18n-interval-6" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 6}"></span>
+  <span id="i18n-interval-7" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 7}"></span>
+  <span id="i18n-interval-10" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 10}"></span>
+
+  <span id="i18n-html" t="[html]html"></span>
+
+  <span id="i18n-prepend-append" t="[prepend]pretest;[append]post-test">Blue</span>
+
+  <span id="i18n-t-vc"> ${'itemWithCount' | t : {count: 10}} </span>
+  <span id="i18n-t-bb"> ${'itemWithCount' & t : {count: 100}} </span>
+
+  <span id="i18n-nf-vc"> ${ 123456789.12 | nf : undefined : locale} </span>
+  <span id="i18n-nf-bb"> ${ 123456789.12 & nf : undefined : 'de'} </span>
+
+  <span id="i18n-nf-vc-cur"> ${ 123456789.12 | nf: {style:'currency', currency: 'EUR' } : 'de' } </span>
+  <span id="i18n-nf-bb-cur"> ${ 123456789.12 & nf: {style:'currency', currency: 'USD' } : locale } </span>
+
+  <span id="i18n-df-vc"> ${ dispatchedOn | df : undefined : locale} </span>
+  <span id="i18n-df-bb"> ${ dispatchedOn & df : undefined : 'de'} </span>
+
+  <span id="i18n-rt-vc"> ${ myDate | rt} </span>
+  <span id="i18n-rt-bb"> ${ myDate & rt} </span>
+
+  <!-- translations via code -->
+  <div>
+    <span id="i18n-code-simple">${translations.simple}</span>
+    <span id="i18n-code-context">${translations.context}</span>
+    <span id="i18n-code-plural">${translations.plural}</span>
+    <span id="i18n-code-interval">${translations.interval}</span>
+    <span id="i18n-code-num">${translations.num}</span>
+    <span id="i18n-code-date">${translations.date}</span>
+    <span id="i18n-code-rtime">${translations.rtime}</span>

```

Comment:
`globalVars` placeholder feature could be tested as well
https://github.com/aurelia/i18n/blob/master/test/unit/feature.placeholders.spec.ts

Reply:
Done